src/app/shared/pipes/currency-default.pipe.ts

```diff
@@ -18,6 +18,6 @@ export class CurrencyDefaultPipe implements PipeTransform {
     digitsInfo = '1.0-2',
     locale = 'en',
   ) {
-    return this.currencyPipe.transform(value / 100, code, display, digitsInfo, locale)
+    return this.currencyPipe.transform(value, code, display, digitsInfo, locale)
```

Comment:
I've tried to change it in all places so the numbers look normal. If you see any big number, it's maybe not divided by 100, so you can do it! 👍🏻